### ISSUE TITLE
Release notes in CHANGELOGs no longer contain links to the PR itself and to the PR author

### ISSUE THREAD

**Issue by ialidzhikov** [kubernetes/kubernetes#106693]
### What happened?

Release notes in CHANGELOGs no longer contain links to the PR itself and to the author.
Open https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#changelog-since-v1223 and verify that the PR and the PR author are not links (just plain text).
Open https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#changelog-since-v1221 and verify that the PR and the PR author are links.

### What did you expect to happen?

The release notes to contain links (not plain text) to the PR itself and to the PR author (as it was previously).

### How can we reproduce it (as minimally and precisely as possible)?

See above.

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by Neilblaze** [comment]
@ialidzhikov Can I go for it?

**Comment by ialidzhikov** [comment]
Let’s wait for input from the sig release folks. They should be able to share whether the change is done on purpose or whether there is a regression with the release note handling.

/cc @cpanato @puerco

**Comment by Neilblaze** [comment]
> Let’s wait for input from the sig release folks. They should be able to share whether the change is done on purpose or whether there is a regression with the release note handling.

Sure! I'll be waiting for their call :D

**Comment by puerco** [comment]
It is a regression, the release notes program was modified to not render the links by default:
https://github.com/kubernetes/release/pull/2274
...but we need to update the release process to render them.  Let's get this fixed before the 1.23 release.
/priotity improtant-soon
